contrib/fuzz/oss_fuzz_build.sh

```diff
@@ -20,6 +20,9 @@ set -x
 
 IFS=$'\n'
 
+INSTRUMENTATION_URL="https://github.com/AdamKorcz/instrumentation"
+COMMIT_HASH="a0052d5b1e2db77ee978890722212230fe86c910" # May 2024 commit
```

Comment:
> Is there more explanation about how the builder will be called differently? How is instrumentation getting there today and what determines the version?

@dmcgowan, the [Dockerfile](https://github.com/google/oss-fuzz/blob/bd6578cae14a1f1ae9a5f5c7ecf18506326468aa/projects/containerd/Dockerfile#L20) clones the repo using the latest commit. 

I've added a variable to store the git commit hash. By using this approach, we can explicitly maintain a specific version of the instrumentation repository. The builder will clone the repository and then checkout the specified commit.

This ensures that we always fetch the required commit instead of cloning the latest version every time.

Reply:
> Is there more explanation about how the builder will be called differently? How is instrumentation getting there today and what determines the version?

Today instrumentation is cloned [from tip, in the `Dockerfile`](https://github.com/google/oss-fuzz/blob/f06c2b532c232b4bfff6ba24720ca15fac6078b4/projects/containerd/Dockerfile#L20) in the oss-fuzz repo.